### ISSUE TITLE
Replacing current form data?

### ISSUE THREAD

**Issue by dzeriho** [dobtco/formbuilder#135]
Can I replace the current form in the FormBuilder with the loaded AJAX response? I'm really having trouble with this. Please help :)

**Comment by tg44** [comment]
I'm having the same problem. Can we modify the form fields from javascript after the form is inicialized?

I'm writing a bridge module to the rapidfire, and my main problem is when a save is run the new fields with cids generates new objects to the database with id-s. I want to write back the indatabase objects id-s to the actual javascript model, becouse when i modify a field the new fields saved again and again.

Is there any proper way to modify the form fields, or reload the form from javascript?

**Comment by SteveChurch** [comment]
@dzeriho Yes, could you explain a little more though, do you mean the initial form which is rendered when your FormBuilder is loaded? - We pull this from a database JSON string and parse it into the FormBuilder
